src/fmripost_aroma/workflows/base.py

```diff
@@ -363,25 +373,96 @@ def init_single_run_wf(bold_file):
         ),
         name='mni6_buffer',
     )
-    if 'bold_mni152nlin6asym' not in functional_cache:
+
+    if ('bold_mni152nlin6asym' not in functional_cache) and ('bold_raw' in functional_cache):
         # Resample to MNI152NLin6Asym:res-2, for ICA-AROMA classification
-        run_stc = (
-            bool(bold_metadata.get('SliceTiming')) and 'slicetiming' not in config.workflow.ignore
```

Comment:
Add this back in.

Reply:
Done in [2f1dc8b](https://github.com/nipreps/fmripost-aroma/pull/58/commits/2f1dc8ba56d80397c297bde25d6e94bb7f73b4a7)